src/app/app.component.ts

```diff
@@ -31,16 +32,24 @@ import { filter } from "rxjs/operators";
       <app-ui></app-ui>
     }`,
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
   configFullscreen: boolean = false;
 
-  constructor(private router: Router) {
+  constructor(
+    private router: Router,
+    private firebaseNotificationService: FirebaseNotificationService,
```

Comment:
Let's move this into a NotificationsModule and do all the registration there, only importing it in AppModule. That makes it easier to cluster and isolate related functionality.
Or is that not loading correctly if we initialize in a Module only?